src/components/tools/table-tool/use-title-size.ts

```diff
@@ -27,8 +28,10 @@ export const useTitleSize = ({ readOnly, columns, dataSet, measureColumnWidth }:
                       (sum, col, i) => sum + (i ? columnWidth(col) : 0),
                       1 - (readOnly ? 0 : kControlsColumnWidth));
     };
+    // eslint-disable-next-line no-unused-expressions
+    rowChanges;
```

Comment:
FYI, you can also use:
```suggestion
    rowChanges; // eslint-disable-line no-unused-expressions
```
which at least reduces the inelegance to a single line. 😉